framework/content/ecl-colors/README.md

```diff
@@ -0,0 +1,16 @@
+# Colors
```

Comment:
Well I put what seems more relevelnt in this context.
If we copy-paste their documentation we will have to maintain it too.
What can be done is simply add a link to their documentation in the readme